src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -24,10 +26,25 @@ const SuggestedOwnerHovercard = ({actor, commits, rules, ...props}) => (
         {actor.id === undefined && (
           <EmailAlert icon="icon-warning-sm" type="warning">
             {tct(
-              'The email [actorEmail]  has no associated Sentry account. Make sure to link alternative emails in [accountSettings:Account Settings].',
+              'The email [actorEmail] is not a member of your organization. [inviteUser:Invite] them or link additional emails in [accountSettings:account settings].',
               {
                 actorEmail: <strong>{actor.email}</strong>,
                 accountSettings: <Link to="/settings/account/emails/" />,
+                inviteUser: (
+                  <a
+                    onClick={() =>
+                      openInviteMembersModal({
+                        initialData: [
+                          {
+                            ...inviteRow,
```

Comment:
I think instead of having to import this `inviteRow` object, can we just merge that in within the component when we convert `initialData` to state